VAMobile/src/screens/HealthScreen/Appointments/AppointmentTypeComponents/SharedComponents/AppointmentCalendarButton.tsx

```diff
@@ -39,6 +39,7 @@ function AppointmentCalendarButton({ appointmentID, attributes, subType, type }:
   const getLocation = (): string => {
     switch (type) {
       case AppointmentDetailsTypeConstants.InPersonVA:
+      case AppointmentDetailsTypeConstants.ClaimExam:
       case AppointmentDetailsTypeConstants.VideoVA:
         if (isIOS() && lat && long) {
           return name || ''
```

Comment:
currently this type doesn't have a title for the calendar. Will need to supply one.

Reply:
Added the calendar event title